### DISCUSSION TITLE
how view desktop mode of website

### DISCUSSION THREAD

**Discussion by lakhininenina** [plateaukao/einkbro#81] [Q&A] [answered]
@plateaukao  
* ver: 8.28.0
i have activated behaviour/ui : desktop mode.

there is no option to toggle desktop version in three dots.

how do i enable desktop version.

**Comment by plateaukao** [comment, marked as answer]
@lakhininenina 
You can long press the three dots icon, it will bring up a quick toggle dialog. And then you'll see a desktop like UI on it. :)
I'll consider add quick toggle dialog to an easier-find place later.

**Reply by lakhininenina** [reply]
> @lakhininenina You can long press the three dots icon, it will bring up a quick toggle dialog. 


@plateaukao unfortunately, it is hit or miss. unreliable, it works sometimes if i close and open app after toggling it.  if possible can you please reboot app when i change the setting.

**Reply by plateaukao** [reply]
@lakhininenina you could click on the reload button to see if it works.
Sometimes if it does not work, that may mean the website already changed url to a mobile site. Then, even if you set style to desktop mode, since it's already mobile site, it won't show the original desktop website, you have to use the original link again.

Rebooting the app after changing desktop mode is not practical since it should be a configuration that works right away; so I may not add this logic into the App.
